.gitattributes

```diff
@@ -0,0 +1,83 @@
+#NOTE: Run `git add : --renormalize` after editing this file.
```

Comment:
Maybe we could remove the ones that will never appear in this project like sass, PHP, sql etc, for simplicity.

Reply:
Yeah, I had the same thought.  Ended-up leaving them all in there;  but I can take out the ones that aren't currently relevant to the project.